velox/common/encode/Base64.h

```diff
@@ -13,16 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 #pragma once
 
 #include <array>
-#include <exception>
```

Comment:
Not sure why, but I'm getting linter error that asks to move these after folly includes:

```
#include <array>
#include <string>

#include "velox/common/base/GTestMacros.h"
```

Any chance you can make that change?

Reply:
Updated the code.